src/test/java/hw8/test/TestScenario.java

```diff
@@ -0,0 +1,38 @@
+package hw8.test;
+
+import hw8.DataProviderHw8;
+import hw8.JdiSite;
+import hw8.beans.MetalsAndColorsTestData;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static hw8.JdiSite.*;
+import static hw8.entities.User.ROMAN;
+import static hw8.pages.MetalAndColorsPage.*;
+
+public class TestScenario implements TestInit {
```

Comment:
What is the reason of 3 tests here instead of ONE ?

Reply:
Because you have provided excel homework file in which expected results are discriped in each step

Reply:
Yep, you're definitely right, but HE consists of only one Test instead. So, you should not mix up two different stuff - Steps and Tests. Step !=Test, you can create Steps with reference to the TC, but you should not create single test for each Step in raw TC from excel file.

Reply:
add precondition steps